lib/engine/corporation.rb

```diff
@@ -37,6 +37,7 @@ def initialize(sym:, name:, **opts)
       end
 
       shares.each { |share| shares_by_corporation[self] << share }
+      @fraction_shares = shares.find { |s| (s.percent % 10).positive? }
```

Comment:
@fractional_shares = shares.any? { |s| (s.percent & 10).positive?

Reply:
Yeah, fractional is a better name so I can change that.
But isn't '%' (modulo) the correct operation here?

Reply:
yes that was a typo